phpstan.neon

```diff
@@ -18,6 +19,10 @@ parameters:
         # ignore PHPUnit data providers return type as they are not checked against the test methods anyway
         - '#^Method PhpCsFixer\\Tests\\.+::provide.+Cases\(\) return type has no value type specified in iterable type (array|iterable)\.$#'
 
+        -
+            message: '#Call to static method .+ with .+ will always evaluate to true.$#'
```

Comment:
can we limit it do current violations only? I don't think we should allow for more `assertTrue(true)`